src/main/java/org/umssdiplo/automationv01/core/managepage/accident/Accident.java

```diff
@@ -27,4 +27,8 @@ public CreateAccident createAccident() {
     public int countAccidents() {
         return CommonEvents.countElement(accidents);
     }
+
+    public String getDescription(int position) {
+        return CommonEvents.getTextContent(accidents.get(position).findElement(By.cssSelector(".description-column")));
```

Comment:
you already have in CommentEvents.findElement(Lis<WebElement> webelement, String lookForText ) you will implement by some especific attribute that you need..